examples/angular/src/app/stoplight-project/stoplight-project.component.ts

```diff
@@ -7,7 +7,7 @@ import { environment } from '../../environments/environment';
   templateUrl: './stoplight-project.component.html',
 })
 export class StoplightProjectComponent {
-  workspaceSlug = 'elements';
-  projectSlug = 'studio-demo';
+  projectId = 'cHJqOjY';
+  platformUrl = 'https://x-6195.stoplight-dev.com';
```

Comment:
Would this be better off called `workspaceUrl`? What you'll see when you go to this URL is a workspace.